server/server.js

```diff
@@ -127,41 +122,19 @@ app.get('/booking/book', async (req, res) => {
   });
 });
 
-//this endpt is triggered when user clicks eligible checkOut date
-app.get('/booking/?campId=&check_in_date=&check_out_date=/bookingTotal', async (req, res) => {
-  // camp_id
-  // Number
-  // check_in_date
-  // Timestamp (ISO 8601)
-  // check_out_date
-  // Timestamp (ISO 8601)
-  // number_nights
-  // Number
-  // total
-  // Number
-  // subTotal
-  // Number
-  // cleaning_fee
-  // Number
-  // weeknight_discount
-  // Number
+//this is a smoke & mirros endpt that should give the illusion of creating a booking event
```

Comment:
Unsure what you mean by this

Reply:
for BMR  I don't need to handle post requests so this route is actually kind of unnecessary and I could have handled everything on the front-end since all the data I need by this pt is already stored in my app state. But I added it now in case I move into advanced content and need to handle post requests ie. actually make a booking and store it in my db